README.md

```diff
@@ -179,6 +179,73 @@ For more details on this signature type, check out the Google Cloud Functions
 documentation on
 [background functions](https://cloud.google.com/functions/docs/writing/background#cloud_pubsub_example).
 
+## Local testing of cloud events
+The setup for cloud functions that accept events is very similar to the instructions in the quickstart, with the following adjustments:
+
+In your package.json, add a signature type (in bold) to your start command:
+<pre>
+  "scripts": {
+    "start": "functions-framework --target=helloWorld <b>--signature-type=event"</b>
+  }
+</pre>
+
+Upon running ```sh npm start ```, you'll see the function is still being served at http://localhost:8080/. However it is no longer accessible via GET requests from the browser. Instead, send a POST request where the request body conforms to the API defined by [push subscriptions](https://cloud.google.com/pubsub/docs/push). 
+
+### Submitting POST request to simulating a pubsub message
+
+Create mockPubsub.json file with the following contents:
+```json
+{
+  "message": {
+    "attributes": {
+      "key": "value"
+    },
+    "data": "SGVsbG8gQ2xvdWQgUHViL1N1YiEgSGVyZSBpcyBteSBtZXNzYWdlIQ==",
+    "messageId": "136969346945"
+  },
+  "subscription": "projects/myproject/subscriptions/mysubscription"
+}
+```
+
+The file can be in any folder on your computer. From the terminal, goto the directory where ```mockPubsub.json``` is located, and run the following command assuming your cloud function is hosted locally on port 8080:
+```sh 
+curl -d "@mockPubsub.json" -X POST http://localhost:8080
```

Comment:
May I also suggest to add the `CE-*` headers, maybe with a _truthy_ value, to the docs so the lib [can know](https://github.com/GoogleCloudPlatform/functions-framework-nodejs/blob/master/src/invoker.ts#L379-L386) that the request is for a Cloud Event?

```sh
curl -d "@mockPubsub.json" -X POST -H "Ce-Type: true" -H "Ce-Specversion: true" -H "Ce-Source: true" -H "Ce-Id: true" http://localhost:8080
```

This would help with issues like #41, and also set the values correctly to the `data` and `context` arguments.